tuning/autotune.py

```diff
@@ -84,7 +109,7 @@ def parse_arguments() -> argparse.Namespace:
         "--devices",
         type=parse_devices,
         default=DEFAULT_DEVICE_LIST,
-        help="Comma-separated list of device IDs (e.g., --devices=0,1). Default: [0]",
+        help="Comma-separated list of device IDs (e.g., --devices=cuda://GPU-xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx,local-sync://default). Default: [0]",
```

Comment:
For now the tuner is really rocm-specific, so this can be
```suggestion
        help="Comma-separated list of device IDs (e.g., --devices=hip://,hip://GPU-UUID). Default: [hip://0]",
```